webrender/src/prim_store.rs

```diff
@@ -625,10 +625,12 @@ impl TextRunPrimitiveCpu {
     }
 
     fn write_gpu_blocks(&self, request: &mut GpuDataRequest) {
-        let mut bg_color = self.font.bg_color;
-        bg_color.a = 0xFF; // not actually affecting the shader
+        let bg_color = ColorF::from(self.font.bg_color);
         request.push(self.font.color);
-        request.push(ColorF::from(bg_color).premultiplied());
+        // this is the only case where we need to provide plain color to GPU
+        request.extend_from_slice(&[
+            GpuBlockData { data: [bg_color.r, bg_color.g, bg_color.b, bg_color.a] }
```

Comment:
I think it would be a good idea to use 1.0 or 0.0 instead of `bg_color.a` here, just to make it clear that we don't care about bg_color's alpha at all.